packages/integrations-hcl/strategies/nomad-pack/load_pack_directory.ts

```diff
@@ -42,13 +42,28 @@ export async function loadNomadPackIntegrationDirectory(
         `For Nomad Packs, the README file must be located at ./README.md, relative to the subdirectory of the Pack.`
     )
   }
-  const readmeContent: string = fs.readFileSync(readmeFile, 'utf8')
+
+  // Read the README
+  const rawReadmeContent: string = fs.readFileSync(readmeFile, 'utf8')
+
+  // If the first line is a H1, remove it. Keeping it will create a
+  // duplicate title in our content on the website.
+  const readmeLines = rawReadmeContent.split('\n')
+  const h1Regex = /^# .*/g
+  if (h1Regex.test(readmeLines[0])) {
+    readmeLines.shift()
+    // The next line is likely an empty line, so let's remove that
+    if (readmeLines[0] === '') {
+      readmeLines.shift()
+    }
+  }
+  const readmeContent = readmeLines.join('\n')
```

Comment:
https://raw.githubusercontent.com/hashicorp/nomad-pack-community-registry/main/packs/aws_efs_csi/README.md

Here's a concrete README that will be processed which will benefit from this parsing logic.